opm/autodiff/SimulatorFullyImplicitBlackoil_impl.hpp

```diff
@@ -483,13 +483,15 @@ namespace Opm
                       const WellMap&    wmap)
         {
             std::vector<int> resv_prod;
-
-            for (int w = 0, nw = wells.number_of_wells; w < nw; ++w) {
-                if (is_resv_prod(wells, w) ||
-                    ((wells.name[w] != 0) &&
-                     is_resv_prod(wmap, wells.name[w], step)))
-                {
-                    resv_prod.push_back(w);
+            if( &wells )
```

Comment:
>>
```C++
const Wells& wells = *((Wells *) 0);
```
>
> Will not that lead to a segfault?

Well, the above is undefined behaviour so anything can happen.  Segmentation faults are the most common outcome.